src/components/Input.tsx

```diff
@@ -0,0 +1,34 @@
+interface UserInputProps {
+    onChange?: (e: any) => void;
```

Comment:
You should never use `any` as type since this will bypass type checking and can cause unexpected bugs.

How about `React.ChangeEvent<HTMLInputElement>`? More on this [here](https://bobbyhadz.com/blog/typescript-react-onchange-event-type)

Reply:
Will take care from next time. I didn't notice this. I have made the changes now.